homeassistant/components/fronius/quality_scale.yaml

```diff
@@ -0,0 +1,79 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
```

Comment:
is missing `data_descriptions` for all flows

Reply:
To be honest, I wouldn't know what to add here. Should this be marked `exempt` then, or should the description be moved?
<img width="501" alt="Bildschirmfoto 2024-11-29 um 11 44 49" src="https://github.com/user-attachments/assets/b784c943-24c3-4d37-b02d-3a6f7a7c241a">

Reply:
You can add "The hostname of your Fronius SolarNet something"

Reply:
done in #132876